src/lib/agent.c

```diff
@@ -96,7 +96,7 @@ void sa_init(struct sof *sof, uint64_t timeout)
 	sof->sa->panic_on_delay = true;
 
 	tr_info(&sa_tr, "sa_init(), ticks = %u, sof->sa->warn_timeout = %u, sof->sa->panic_timeout = %u",
-		ticks, sof->sa->warn_timeout, sof->sa->panic_timeout);
+		(int)ticks, (int)sof->sa->warn_timeout, (int)sof->sa->panic_timeout);
```

Comment:
All the Xtensa architecture variants are 32-bit so int is 32-bit. I think you'll truncate values. Whether that is acceptable or not I'm not sure.

Reply:
The reason why tr_info doesn't support 64-bit arguments is because ANY arguments you give are turned into up to 4 integers which are stored as-is in the trace structure. 32-bit integers, to be clear.